test-data/unit/fixtures/primitives.pyi

```diff
@@ -20,4 +20,5 @@ class str:
 class bytes: pass
 class bytearray: pass
 class tuple(Generic[T]): pass
+class list(Generic[T]): pass
```

Comment:
The header of this file says `# builtins stub with non-generic primitive types`, so I would avoid adding generic classes here. We have already enough fixtures with `list`, why do you need more?